quipucords/api/credential/tests_credential.py

```diff
@@ -59,18 +59,18 @@ def create(self, data):
     def create_expect_400(self, data):
         """We will do a lot of create tests that expect HTTP 400s."""
         response = self.create(data)
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        assert response.status_code == status.HTTP_400_BAD_REQUEST
```

Comment:
Since these are in a Django `TestCase`, I would assume `self.assertEqual` (and similar methods) provides at least some useful syntactic sugar. Are we deliberately refactoring these Django-specific assert* calls to plain Python `assert` statements in general?

Reply:
there's a huge difference on how the error is formatted!
I have an example in [this commit](6c657c949e7c09b07bd998a868d0eaa669b6b663)

essentially, plain asserts are not treated in a special way by pytest, with custom formatters that will help spot differences in long diffs  - and they are affected by pytest -v flag.

unittest asserts, on the other hand, don't show the whole diff (at least not when invoked by pytest) and don't show nothing interesting.

Reply:
TBH I'd replace all unit test asserts with plain asserts. Everytime I need to debug I start replacing.

(there's nothing broken now, but while developing I had to debug it)

Reply:
Interesting. I hadn't experienced this kind of problem before because every other Django project I worked on used the Django test runner, not a third-party one like pytest. Thank you for providing context. I think I'm okay with this!